SCYLLA-VERSION-GEN

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 PRODUCT=scylla
 VERSION=666.development
```

Comment:
What about `VERSION` ? i guess we should start updating this value to match Scylla repo, right?

Reply:
`VERSION` is for define package version.

We use 666.development for master/next branch, and it need to be update to next release version when we create abranch or a tag, something like:
https://github.com/scylladb/scylla-jmx/commit/77d6f5e3e7d5e9be83df86e35c695c7bd8d8748e

When we specify VERSION to 5.0.2, build_{reloc,rpm,deb}.sh will automatically versioned like
scylla-jmx-5.0.2-20240611.d0cd907 ($VERSION-$DATE.$HASH).

> i guess we should start updating this value to match Scylla repo, right?

I think we do not need to match version with Scylla repo, it was required since our metapackage requires exact version number, like:
```
Requires:       %{product}-jmx = %{version}-%{release}
```
https://github.com/scylladb/scylladb/blob/branch-5.4/dist/redhat/scylla.spec#L14C1-L15C1

It was needed to install specific version of Scylla using the metapackage.
But now we dropped scylla-jmx dependency, I think now we can version scylla-jmx independently.

Note that even we can versioning independently, we need to keep increment from existing version. 
And latest available version is probably "scylla-jmx-6.0.0-0.20240606.a77615adf324":
https://downloads.scylladb.com/downloads/scylla/rpm/centos/scylladb-6.0/noarch/

So we can start from 6.0.0-0.<more recent date> (or 6.0.1 to make versioning cleaer).